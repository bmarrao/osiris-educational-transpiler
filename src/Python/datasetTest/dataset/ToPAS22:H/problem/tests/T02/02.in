5
2 0
a A
2 1
a A
A a
2 1
a A
a A
2 0
IVSB IWgLD
2 0
WFG oN
