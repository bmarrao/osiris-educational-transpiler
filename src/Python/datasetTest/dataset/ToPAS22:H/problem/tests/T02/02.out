Possivel: 2
Impossivel
Impossivel
Possivel: 2
Possivel: 2
